pkgs/applications/misc/plasma-applet-volumewin7mixer/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, cmake, extra-cmake-modules, plasma-framework, kwindowsystem, fetchFromGitHub, }:
+
+let version = "23"; in
```

Comment:
Most packages integrate the `version` attribute into the attribute set passed to `mkDerivation`. This can be done by writing `rec` before the attribute set.

    stdenv.mkDerivation rec {
      name = "plasma-applet-volumewin7mixer-${version}";
      version = "23";

Also you should interpolate the version into the name.

Reply:
Fixed ;)